packages/common/README/HOOKS.md

```diff
@@ -137,16 +137,22 @@ function TestComponent() {
 Custom hook used to get a translation function tObj that translates TranslationObjects.
 
 ### usePreventExit
-```tsx
+```ts
 import { usePreventExit } from '@monkvision/common';
 
-function TestComponent() {
-  usePreventExit();
-  return <>...</>;
+function MyComponent() {
+ const { allowRedirect } = usePreventExit(true);
+ function anyEvent(){
+  allowRedirect();
+  /** ... */
+ }
 }
 ```
+
 This hook is used to prevent the user from leaving the page by displaying a confirmation dialog when the user tries to
 leave the page.
```

Comment:
Add after the `.` : `It also provides a callback that can be called to manually re-enable the direct exit.`